packages/features/ee/payments/components/Payment.tsx

```diff
@@ -70,22 +78,33 @@ export default function PaymentComponent(props: Props) {
     const card = elements.getElement(CardElement);
     if (!card) return;
     setState({ status: "processing" });
-    const payload = await stripe.confirmCardPayment(props.payment.data.client_secret!, {
-      payment_method: {
-        card,
-      },
-    });
-    if (payload.error) {
+
+    let payload;
```

Comment:
Why do we rely on webhooks for post purchase logic? Couldn't we pass the payload to a trpc endpoint?

Reply:
Not needed, in this situation it only complicates matters.